q2_phylogeny/tests/test_raxml.py

```diff
@@ -20,10 +24,26 @@
                                  _set_raxml_version)
 
 
+@pytest.fixture(scope='class')
+def data_dir(request, tmpdir_factory):
+    """Copies test data to `data_dir` and defines it as an attribute on
+    `RaxmlTests`."""
+    src = pkg_resources.resource_filename(request.cls.package, 'data')
+    dst = str(tmpdir_factory.mktemp('raxml_tests').join('data'))
+    shutil.copytree(src, dst)
+    request.cls.data_dir = dst
+
+
+@pytest.mark.usefixtures('data_dir')
```

Comment:
This avoids overriding `setUp` altogether and runs just once for the whole class.

Reply:
Cool! At that point though maybe it would be best to use `setupClass` hook, instead? That way you aren't locking yourself into pytest machinery. Generally, if it works with `unittest`, it'll work with any test runner.